builtin/providers/vcd/resource_vcd_dnat.go

```diff
@@ -49,6 +55,10 @@ func resourceVcdDNATCreate(d *schema.ResourceData, meta interface{}) error {
 	vcdClient.Mutex.Lock()
 	defer vcdClient.Mutex.Unlock()
 	portString := getPortString(d.Get("port").(int))
+	trlateportString := portString // default
```

Comment:
Can we make this variable a little easier to read? `translatedPortString` isn't much longer.